contribute/documentation.md

```diff
@@ -6,9 +6,9 @@ title: Documentation Contributions
 
 There are several ways you can help out with the improvement of Scala documentation. These include:
 
+* API Documentation in Scaladoc
 * Code examples and tutorials in activator templates.
 * The Scala Wiki
```

Comment:
I'd remove the link to the Scala Wiki altogether. I think it's not really updated anymore. Whatever is interesting in it, Google indexes. I wouldn't be directing contribution energy there though.